### DISCUSSION TITLE
Is there any way to manually prune stale options?

### DISCUSSION THREAD

**Discussion by BrendanMesters** [ajeetdsouza/zoxide#822] [Q&A] [answered]
I currently have two directories named `python` which are both used for a similar data manipulation, just on different projects. 
One of the two projects is done now, but as I used that directory often it automatically redirects me to that directory.

So I was wondering if there is an easy way to prune depricate items in the lookup table of ZOxide?

**Comment by ajeetdsouza** [comment, marked as answer]
Does `zoxide remove <PATH>` work?

**Reply by BrendanMesters** [reply]
thanks a bunch, that was indeed the command I was looking for :-)
